gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagManagementDagActionStoreChangeMonitor.java

```diff
@@ -52,7 +52,7 @@ public DagManagementDagActionStoreChangeMonitor(Config config, int numThreads,
   }
 
   /**
-   * This implementation passes on the {@link DagActionStore.DagAction} to the
+   * This implementation passes on {@link DagActionStore.DagAction} to the
```

Comment:
"This implementation passes on *the* {@link DagActionStore.DagAction} to -t-h-e- {@link DagManagement} instead..."

(strike `-t-h-e-`)